SingleViewApi.Tests/V1/UseCase/GetCustomerByIdUseCaseTests.cs

```diff
@@ -137,7 +137,7 @@ public void GetsACustomerFromPersonApiJigsawAndAcademyCouncilTax()
                 Title = null,
                 FirstName = mockFirstName,
                 Surname = mockLastName,
-                ContactDetails = fakeJigsawContactDetails,
+                // ContactDetails = fakeJigsawContactDetails,
```

Comment:
Can this be either uncommented or removed?

Reply:
whoops